docs/spec/src/protocol/payments/payment_system.md

```diff
@@ -0,0 +1,179 @@
+- Payment usage is not tracked on-chain; instead, the EigenDA Disperser maintains a DynamoDB table recording total
+historical usage for all clients.
+- When processing a dispersal, the Disperser compares a user's total historical usage against their on-chain deposits
+in the PaymentVault to determine if they have sufficient funds.
+- Clients fetch the latest cumulative payment state from the EigenDA Disperser on startup via the `GetPaymentState` RPC.
+
+#### 2.2.1 Why Only the EigenDA Disperser?
+
+- On-demand payments are supported only through the EigenDA Disperser.
+- Since EigenDA currently lacks a consensus mechanism, validators cannot easily coordinate to limit total on-demand
+throughput across the network.
+- Therefore, the EigenDA Disperser fills the role of arbiter, ensuring that total network throughput doesn't exceed
+configured levels.
+
+#### 2.2.2 Cumulative Payment
+
+The cumulative payment is a field set in the PaymentHeader by the client when making a dispersal. It represents the
+total cost (in wei) of all previous dispersals, plus the new dispersal.
+
+- **Historical context:** In a prior payments implementation, the cumulative payment field included by the client in
+  the PaymentHeader had to be monotonically increasing, and the disperser would verify that each new cumulative payment
+  received exceeded the previous one by at least the cost of the new blob. This severely limited concurrency, since
+  clients had to make sure that all on-demand dispersals were handled by the disperser in strict order. In practice,
+  that meant waiting for the entire network roundtrip, for dispersal N to be confirmed before submitting dispersal N+1.
+- **Current implementation:** The system has been simplified to improve concurrency:
+   - Clients still populate the `cumulative_payment` field with their local calculation of cumulative payment.
+   - However, the Disperser now only checks if this field is non-zero (to determine payment type) and ignores the exact
+   value.
+   - The Disperser tracks each account's on-demand usage in DynamoDB, incrementing by the blob cost for each dispersal.
+   - This removes the strict ordering requirement and allows for highly concurrent dispersals.
+- **Why clients still populate the field:** Although currently unused beyond the zero/non-zero check, clients continue
+  to populate this field with meaningful values. This preserves the option to reintroduce cumulative payment validation
+  in the future if needed.
+
+## 3. Client Payment Strategy
+
+### 3.1 Payment Header
+
+Each dispersal request includes a [PaymentHeader](../../../../../api/proto/common/v2/common_v2.proto#L111) containing:
+- `account_id`: Ethereum address identifying the payment account
+- `timestamp`: Nanosecond UNIX timestamp (serves as nonce)
+- `cumulative_payment`: Variable-length big-endian uint for on-demand dispersal (or empty for reservation dispersal)
```

Comment:
nit: prob good to specify that its at most a uint256, since the source of truth is found onchain.